2021/day14/day14.java

```diff
@@ -0,0 +1,89 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.io.File;
+
+public class day14 {
+    public static class Rules {
+        String pair;
+        String single;
+    }
```

Comment:
I would call it a `Rule`, it's not like this represents multiple of them. Also, to make this more readable, I would rename `single` to `outcome` or `result` or something along these lines.